FlowCrypt/Functionality/Services/AppStartup.swift

```diff
@@ -68,8 +68,13 @@ struct AppStartup {
 
         switch entryPoint {
         case .mainFlow:
-            let contentViewController = InboxViewContainerController(appContext: appContext)
-            viewController = SideMenuNavigationController(appContext: appContext, contentViewController: contentViewController)
+            guard let user = appContext.dataService.currentUser else { return }
```

Comment:
Should not be possible, so I just return. Other ideas are welcome

Reply:
Could it be refactored so that user is somehow an attribute of mainFlow? Like case .mainFlow(let user). That would be best. That way a main flow truly cannot be initiated without user.